### PR TITLE
STY: Refactor Python classes to be able to reuse code for ROC curve

### PR DESCRIPTION
**Pull request by stephanecollot**

### CODE REVIEW

File: mmu/methods/prcurve.py [hunk 1]
```diff
@@ -87,24 +72,33 @@ class PrecisionRecallCurveUncertainty:
...
-            "mult": {"mult", "multinomial"},
-            "bvn": {"bvn", "bivariate", "elliptical"},
-        }
+        BaseCurveUncertainty.__init__(self)
+
+        # Option 1
+        self.bvn_grid_curve_error_mt_func = pr_bvn_grid_curve_error_mt
+        self.bvn_grid_curve_error_func = pr_bvn_grid_curve_error
+        self.multn_grid_curve_error_mt_func = pr_multn_grid_curve_error_mt
+        self.multn_grid_curve_error_func = pr_multn_grid_curve_error
```
Comment (stephanecollot) [inline mmu/methods/prcurve.py]: Either we need to give those as parameters

Comment (stephanecollot) [reply mmu/methods/prcurve.py]: Either we need to redefine
_compute_bvn_scores
_compute_multn_scores

**Comment by RUrlus** [comment]
Ha @stephanecollot, I am assuming the draft status can be removed?

**Comment by stephanecollot** [comment]
Yes, could you it? I cannot from my mobile.

**Comment by RUrlus** [comment]
Closing as superseded by #46

**Pull request closed without merging**